### ISSUE TITLE
Phrases in other languages need markup

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4218]
We are not currently compliant with [WCAG 3.1.2 Language of Parts](https://www.w3.org/TR/WCAG22/#language-of-parts)
> The human language of each passage or phrase in the content can be programmatically determined except for proper names, technical terms, words of indeterminate language, and words or phrases that have become part of the vernacular of the immediately surrounding text.

We do currently set hte language of the page (as per WCAG 3.1.1) but where words or phrases are not in that language, they need to be [marked up as per technique H58.](https://www.w3.org/WAI/WCAG22/Techniques/html/H58.html).  We do not need to do this with words in common usage, such as `in situ` but for clarity, we should consider maintaining a list of words not being marked up.

The following examples from the #4194 should be considered when implementing this feature.
- Sample 2, items 39 and 54
- Sample 3, items 20 and 35

**Comment by StephDriver** [comment]
Can we implement an extension for TinyMCE along these lines? https://github.com/openedx/tinymce-language-selector

**Comment by StephDriver** [comment]
at implementation need to pick out the correct tool - the above is an example.

**Comment by joemull** [comment]
This is quite closely related to https://github.com/openlibhums/janeway/issues/4139

**Comment by joemull** [comment]
I think we shouldn't install this TinyMCE plugin before we have JS package management in place.

Blocked by https://github.com/openlibhums/devops/issues/2.

**Comment by StephDriver** [comment]
meanwhile, implement and test this solution.
